peering/migrations/0071_auto_20210321_1843.py

```diff
@@ -6,7 +6,10 @@
 
 class Migration(migrations.Migration):
 
-    dependencies = [("peering", "0070_remove_router_last_deployment_id")]
+    dependencies = [
+        ("net", "__first__"),
```

Comment:
I'd prefer having the actual migration name instead of an abstract reference to it. `0001_initial` instead of ` __first__`.